packages/commonwealth/client/scripts/views/modals/WelcomeOnboardModal/steps/PersonalInformationStep/PersonalInformationStep.tsx

```diff
@@ -22,7 +22,7 @@ import {
   CWFormRef,
 } from 'views/components/component_kit/new_designs/CWForm';
 import { CWTextInput } from 'views/components/component_kit/new_designs/CWTextInput';
-import useNotificationSettings from 'views/pages/notification_settings/useNotificationSettings';
+import useNotificationSettings from 'views/pages/NotificationSettingsOld/useNotificationSettings';
```

Comment:
I think we should use the newer notification settings in this component, or use different notification settings config depending on `FLAG_KNOCK_INTEGRATION_ENABLED`.

Else when `FLAG_KNOCK_INTEGRATION_ENABLED` is enabled in prod, this component would give undesired results

Reply:
> You mean just another flag? I'm trying to avoid creating too many. I think we're at 5-6 so far.

Actually no, I meant that the `PersonalInformationForm` uses the old way to subscribe user to notifications, once knock is enabled in app we would want to use knock notification (ex: `trpc.subscription.createCommunityAlert` and others) instead of the old notifications in this component. Otherwise this component will break/give-undesired-results when knock is enabled

Reply:
This component won't be rendered and we will use the new notifications page which uses tRPC. The old page can be removed at that point. 

I think we're good here.

Reply:
Actually `PersonalInformationForm` is rendered when a user creates a new account while `FLAG_USER_ONBOARDING_ENABLED` is `true`.

The said I think this will cause a bug is because `PersonalInformationForm` uses the `useNotificationSettings` from old notifications page which enables/disables notifications using old older `app.user.notifications.` methods

Reply:
Where do you see PersonalInformationForm?  I don't have that as a symbol in my entire repo except for personalInformationFormValidation

Reply:
Ah mb, sry for the confusion, the component name is `PersonalInformationStep.tsx`

cc: @burtonator

Reply:
can we get another look on this, please @burtonator